starlette_admin/statics/js/render.js

```diff
@@ -111,4 +111,20 @@ const render = {
       )
       .join("")}</div>`;
   },
+  interval: function render(data, type, full, meta, fieldOptions) {
+    if (!data) return null_column();
+    if (Array.isArray(data) && data.length == 0) return empty_column();
+    data = Array.isArray(data) ? data : [data];
+    if (type != "display") return data.map((d) => new URL(d.url));
```

Comment:
This should be a string representation of the data 
```suggestion
    if (type != "display") return data.map((d) => );
```